ntbs-service/Pages/Notifications/Edit/_SaveButton.cshtml

```diff
@@ -1,13 +1,13 @@
-@model ntbs_service.Models.Notification
-
-@{
-    var buttonName = Model?.NotificationStatus == NotificationStatus.Draft ? "Save and continue" : "Save";
-}
-<button
-    nhs-button-type="Standard"
-    classes="ntbsuk-button--primary"
-    name="actionName"
-    value="Save"
-    id="save-button">
-    @buttonName
+@model ntbs_service.Models.Notification
+
+@{
+    var buttonName = Model?.NotificationStatus == NotificationStatus.Draft ? "Save and continue" : "Save";
+}
+<button
+    nhs-button-type="Standard"
+    classes="ntbsuk-button--primary"
+    name="actionName"
+    value="Save"
+    id="save-button">
```

Comment:
This will result in a duplicate id - not sure if we mind too much though?

Reply:
I did think that, and checked that it's not a problem accessibility-wise, but it seems fine from that point of view. it might crop up in testing, so i'm double checking that now

Reply:
With the firmness of the responses here: I think it's safe to call it 'not good practice':
https://stackoverflow.com/questions/22059438/html-duplicated-id

Reply:
Yup, very fair - and it was making the tests awkward. fix just pushed :)